smarts/core/tests/test_renderers.py

```diff
@@ -109,7 +116,12 @@ def test_optional_renderer(smarts, scenario):
     assert not smarts.is_rendering
     for _ in range(10):
         smarts.step({})
-    renderer = smarts.renderer
+
+    try:
+        renderer = smarts.renderer
+    except Exception as e:
+        raise RendererException.required_to("run test_renderer.py")
```

Comment:
I do not think this does as you expect. The smarts.renderer does not throw an exception.

```suggestion
    renderer = smarts.renderer
    if not renderer:
        raise RendererException.required_to("run test_renderer.py")
```

Reply:
![image](https://user-images.githubusercontent.com/47334253/126546273-46e18376-7d37-4e3d-8670-06b08351cd5e.png)
I tested it and it was showing the Exception and the warning for me in stdout. It appears to be working for me.